syncopy/tests/backend/test_fooofspy.py

```diff
@@ -0,0 +1,143 @@
+# -*- coding: utf-8 -*-
+#
+# syncopy.specest fooof backend tests
+#
+import numpy as np
+import pytest
+
+from syncopy.specest.fooofspy import fooofspy
+from fooof.sim.gen import gen_power_spectrum
+from fooof.sim.utils import set_random_seed
+
+from syncopy.shared.errors import SPYValueError
+import matplotlib.pyplot as plt
+
+
+def _power_spectrum(freq_range=[3, 40], freq_res=0.5, periodic_params=[[10, 0.2, 1.25], [30, 0.15, 2]], aperiodic_params=[1, 1]):
+    """
+     aperiodic_params = [1, 1]  # use len 2 for fixed, 3 for knee. order is: offset, (knee), exponent.
+     periodic_params = [[10, 0.2, 1.25], [30, 0.15, 2]] # the Gaussians: Mean (Center Frequency), height (Power), and standard deviation (Bandwidth).
+    """
+    set_random_seed(21)
+    noise_level = 0.005
+    freqs, powers = gen_power_spectrum(freq_range, aperiodic_params,
+                                       periodic_params, nlv=noise_level, freq_res=freq_res)
+    return freqs, powers
+
+
+class TestSpfooof():
+
+    freqs, powers = _power_spectrum()
+
+    def test_spfooof_output_fooof_single_channel(self, freqs=freqs, powers=powers):
```

Comment:
I think we can get rid of at least the `_spfooof_` in each and every test function name, pytest anyways will list these with the clas name `TestFooofSpy`preprended
EDIT: I did that now, maybe you can do this for the frontend tests as well?

Reply:
true, done in 0eeffbc